build/package/mac_notarize.sh

```diff
@@ -22,17 +22,36 @@ set -Eeou pipefail
 
 # this script could run in parallel for both x86_64 and arm64
 # we need to make sure to call the right one at the right time
+
+curl "${NOTARY_SERVICE_URL}" --output macos-notary.zip
+unzip -u macos-notary.zip
+chmod 755 ./darwin_amd64/macnotary
+
 if [[ -f "./dist/macos_darwin_amd64/bin/mongocli" && ! -f "./dist/mongocli_macos_signed_x86_64.zip" ]]; then
   echo "notarizing x86_64"
-  gon -log-level=error gon_x86_64.json
+  zip -rj ./dist/macos_darwin_amd64/mongocli_amd64_bin.zip ./dist/macos_darwin_amd64/bin/mongocli # The Notarization Service takes an archive as input
```

Comment:
is it possible then to zip both the arm and amd64 bins and runa single notarization process?

Reply:
working on it

Reply:
Updated